infrastructure/code/code_html.go

```diff
@@ -46,6 +47,42 @@ func getDataFlowHeadingHtml(issue snyk.CodeIssueData) string {
 	return fmt.Sprintf("Data Flow - %d %s", dataFlowCount, stepWord)
 }
 
+func getIgnoreDetailsHtml(isIgnored bool, ignoreDetails *snyk.IgnoreDetails) (ignoreDetailsHtml string, visibilityClass string) {
+	if !isIgnored {
+		return "", "hidden"
+	}
+
+	labels := []string{"Category", "Ignored On", "Expiration", "Ignored By"}
+
+	detailsMap := map[string]string{
+		"Category":   ignoreDetails.Category,
+		"Ignored On": formatDate(ignoreDetails.IgnoredOn),
+		"Expiration": ignoreDetails.Expiration,
+		"Ignored By": ignoreDetails.IgnoredBy,
+	}
+
+	pairs := `<div class="ignore-details-wrapper">`
+	for _, label := range labels {
+		value := detailsMap[label]
+		pairs += fmt.Sprintf(`<div class="ignore-details-row">
+		<div class="ignore-details-label">%s</div>
+		<div class="ignore-details-value">%s</div>
+		</div>`, label, value)
+	}
+	pairs += `</div>`
+
+	reason := fmt.Sprintf(`<div class="ignore-details-reason-wrapper">
+		<div class="ignore-details-reason-label">Reason</div>
+		<div class="ignore-details-reason-text">%s</div>
+	</div>`, ignoreDetails.Reason)
+
+	warning := `<div class="ignore-next-step-text">Ignores are currently managed in the Snyk web app.
+		To edit or remove the ignore please go to: <a class="styled-link" href="https://app.snyk.io" target="_blank" rel="noopener noreferrer" >https://app.snyk.io</a>.</div>`
```

Comment:
This should use the actual endpoint webui address from config.CurrentConfig()

Reply:
Yup we plan on adding a ticket as part of manual testing! See https://github.com/snyk/snyk-ls/pull/483#discussion_r1559178885